### ISSUE TITLE
why is everything being cloned to the $HOME folder?

### ISSUE THREAD

**Issue by bckelley** [ChrisTitusTech/linutil#161]
**Is your feature request related to a problem? Please describe.**
seems to be cleaner to have a central location `e.g. .github/` 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by sevu11** [comment]
@bckelley please fill out the form correctly. This issue contain no information.

**Comment by bckelley** [comment]
What part of that wasn't concise?

**Comment by nnyyxxxx** [comment]
Nothing is being cloned to the home folder when running
```bash
curl -fsSL https://christitus.com/linux | sh
```
If you are running this
```bash
git clone https://github.com/ChrisTitusTech/linutil/
```
Then it's not the repos fault that it's being cloned to the home folder. You can specify a location for the cloned repo to be dropped in though for instance
```bash
git clone https://github.com/ChrisTitusTech/linutil/ $HOME/Downloads/linutil
```
or
```bash
git clone https://github.com/ChrisTitusTech/linutil/ $HOME/.github/linutil
```
@bckelley

**Comment by bckelley** [comment]
Sorry maybe I wasn’t clear I was referring to the 1-compile-setup.sh which does indeed create a linutil directory under $HOME. I think it to be bad practice to clutter up the home directory to got repos!On Aug 23, 2024, at 03:26, nyx ***@***.***> wrote:﻿
Nothing is being cloned to the home folder when running
curl -fsSL https://christitus.com/linux | sh
If you are running this
git clone https://github.com/ChrisTitusTech/linutil/
Then it's not the repos fault that it's being cloned to the home folder. You can specify a location for the cloned repo to be dropped in though for instance
git clone https://github.com/ChrisTitusTech/linutil/ $HOME/Downloads/linutil
or
git clone https://github.com/ChrisTitusTech/linutil/ $HOME/.github/linutil
@bckelley

**Comment by nnyyxxxx** [comment]
Yeah, I considered you meant that. IMO there is nothing wrong with having one folder in the $HOME dir for a git project. Though it may be up to personal preference. @bckelley

**Comment by bckelley** [comment]
I usually do like $HOME/.github/${repo_name} I seeing the script didn’t even have a simple if statement to look for a passed argument for a directory was off putting and I don’t understand shell that well to be that complex in my scripting for a commitOn Aug 23, 2024, at 11:33, Nyx ***@***.***> wrote:﻿
Yeah, I considered you meant that. IMO there is nothing wrong with having one folder in the $HOME dir for a git project. Though it may be up to personal preference. @bckelley

**Comment by nnyyxxxx** [comment]
Solved with #195 @bckelley

**Comment by bckelley** [comment]
@nnyyxxxx approved
